mergebot_poller.py

```diff
@@ -105,7 +105,7 @@ def search_github_pr(self, pull):
         """
         # Load comments for each pull request
         comments = pull.fetch_comments()
-        if comments == []:
+        if not comments:
             print_flush('No comments. Moving on.')
```

Comment:
For debugging may want to say: "No comments on PR {}, moving on." (so that we know what we're moving on from)